packages/core/src/components/radio-button/radio-button.stories.js

```diff
@@ -35,3 +36,11 @@ export const SelectedDisabled = () => `
     </div>
 `;
 SelectedDisabled.storyName = 'Selected & disabled';
+
+export const WithHelperText = () => `
+    <div class="hds-radio-button">
+      <input type="radio" id="radio3" class="hds-radio-button__input" name="example" value="baz" disabled />
+      ${getLabel('radio5')}
+      ${getHelperText()}
```

Comment:
This has a similar accessibility issue with the missing aria-describedby than Checkbox